pkg/spec/gpyrpc/gpyrpc.proto

```diff
@@ -479,6 +479,7 @@ message KclType {
 	string pkg_path = 13;                // `pkg_path` represents the path name of the package where the attribute is located.
 	string description = 14;             // `description` represents the document of the attribute.
 	map<string, Example> examples = 15;  // A map object to hold examples, the key is the example name.
+	map<string, KclType> referencedBy = 16;          // contains referenced by info for schema
```

Comment:
Why add referenceBy here? It should be same as the spec: https://github.com/kcl-lang/kcl/blob/main/kclvm/spec/gpyrpc/gpyrpc.proto#L482